packages/router/router.ts

```diff
@@ -2442,6 +2464,10 @@ export function createStaticHandler(
     return {
       ...context,
       matches,
+      activeDeferreds:
+        activeDeferreds.size > 0
+          ? Object.fromEntries(activeDeferreds.entries())
+          : null,
```

Comment:
Do we _need_ to convert this to an object?  Or could it remain a `Map` since it's never intended to be serialized?